modules/oe_content_publication/config/install/field.field.node.oe_publication.oe_teaser.yml

```diff
@@ -13,7 +13,7 @@ bundle: oe_publication
 label: Teaser
 description: 'A short overview of the information on this page. The teaser will be displayed in list views and search engine results, not on the page itself. Limited to 150 characters for SEO purposes.'
 required: true
-translatable: true
+translatable: false
```

Comment:
Let's not update the field translatablility for existing fields, we have a dedicated issue for that: EWPP-169.

Reply:
This is reset.